manager/testdata/unittests/m4dcopyapp-csv.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   appInfo:
     intent: fraud-detection
   data:
-  - dataSetID: '{"asset_id": "xxx", "catalog_id": "s3-csv"}'
+  - dataSetID: "s3-csv/allow-dataset"
```

Comment:
This is not the same test. The original is equivalent to "s3-csv/xxx" or "s3-csv/redact-dataset".

Reply:
Will change, but how is that reflected in the test code? (Given that both pass the test)

Reply:
Got it. Changed according to your suggestion in case the test will need it in the future.